apim-publisher/k8s/apim-config.yaml

```diff
@@ -0,0 +1,109 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: apim-config
+  namespace: apim-be-dev
+data:
+  apiInformation.json: |
```

Comment:
I'm not sure we need the ConfigMap data here - I believe we'd discussed in the design sessions and ADR that the whole /apimartifacts folder will be copied in and applied. The updates to apiInformation.json, policy.xml, specification.yaml, etc. will happen within /apimartifacts prior to it being copied into the publisher container.

Reply:
This is an artifact folder mounted from configmam

Reply:
Why do we need apiInformation.json, policy.xml, specification.yaml, etc. in ConfigMap data if the same file data will already be present in the /apimartifacts folder?